include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -541,10 +552,10 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 									continue;
 
 								auto sample = lineBuffer+i*MaxChannels;
-								value_type swizzledSample[MaxChannels];
+								value_type preSwizzleSample[MaxChannels];
 
 								// TODO: make sure there is no leak due to MaxChannels!
-								base_t::template onDecode(inFormat, state, srcPix, sample, swizzledSample, inBlockCoord.x, inBlockCoord.y);
+								base_t::template onDecode(inFormat, state, srcPix, preSwizzleSample, sample, inBlockCoord.x, inBlockCoord.y);
```

Comment:
@achalpandeyy before the `swizzledSample` wouldn't get used at all which would mess with Derivative Map from Heightmap generation